src/client/components/ConsentsForm.tsx

```diff
@@ -5,8 +5,6 @@ import useClientState from '@/client/lib/hooks/useClientState';
 import { onboardingFormSubmitOphanTracking } from '@/client/lib/consentsTracking';
 import { CsrfFormField } from '@/client/components/CsrfFormField';
 import { buildUrlWithQueryParams } from '@/shared/lib/routeUtils';
-import { CmpConsentedStateHiddenInput } from '@/client/components/CmpConsentStateHiddenInput';
-import { useCmpConsent } from '@/client/lib/hooks/useCmpConsent';
```

Comment:
I'm assuming `useCmpConsent` is used somewhere else, which is why it hasn't been removed from this PR?

Reply:
Yeah, it's used to determine whether the ad personalization permission gets shown on the onboarding data page.